react/week-2/lesson.md

```diff
@@ -443,6 +443,14 @@ function FruitCounter() {
 }
 ```
 
+
+| **Exercise C** |
+| :--- |
+| 1. Open the `pokedex` React application and open the `CaughtPokemon.js` file. |
+| 2. Create a new state variable called `totalCaught` and initialise it to `0` |
+| 3. Add a button to the component with an `onClick` handler which increments the `totalCaught` count. Refer to the examples above to help you. |
```

Comment:
I wonder if we could add some prediction here? Something simple like "before you click the button discuss with your classmates what will happen".

Reply:
I've split this exercise into exercises C and D